oviewer/oviewer.go

```diff
@@ -700,6 +706,13 @@ func (root *Root) prepareRun(ctx context.Context) error {
 		root.Screen.EnableMouse(MouseFlags)
 	}
 
+	if RawF {
+		root.General.Converter = convRaw
+	}
+	if AlignF {
+		root.General.Converter = convAlign
+	}
```

Comment:
Maybe using

https://pkg.go.dev/github.com/spf13/pflag#Set

So set the converter flag?

Reply:
Thank you for your comment.
I moved to the main side and fixed it.